Source/DafnyLanguageServer/DafnyRequestNames.cs

```diff
@@ -4,5 +4,6 @@ public static class DafnyRequestNames {
     public const string CompilationStatus = "dafny/compilation/status";
     public const string GhostDiagnostics = "dafny/ghost/diagnostics";
     public const string VerificationStatusGutter = "dafny/verification/status/gutter";
+    public const string VerificationStatusNotification = "dafny/textDocument/verificationStatus";
```

Comment:
> I've seen the APIs above look more like ...

To me it looks like the current code took the name of the variable and then replaced camel-casing with `/` separation to form the operation name, which I find strange. We can use camel-casing in the operation name without introducing additional `/`'s, just like LSP does.

LSP categorises the operations based on what they operate on in the IDE, the main ones are `textDocument/<name>`, `workspace/<name` and `window/<name>`. I think we should stick to that.

I'm guessing `dafny` and `verification` always occur together, because all the APIs we are adding relate to verification. APIs that don't relate to verification should already be in LSP.

I suggest we use:

```c#
    public const string Counterexample = "textDocument/dafny/counterexample";
    public const string GhostDiagnostics = "textDocument/dafny/ghostness";
    public const string VerificationStatusGutter = "textDocument/dafny/lineStatus";
    public const string VerificationStatusNotification = "textDocument/dafny/symbolStatus";

    // Will be removed
    public const string CompilationStatus = "dafny/compilation/status";
```